pyro/distributions/torch.py

```diff
@@ -2,6 +2,7 @@
 
 import torch
 
+from pyro.distributions.binomial import Binomial  # noqa: F403
```

Comment:
It seems weird to double inherit from `TorchDistributionMixin`. I think cleaner approaches would be to either
1. move `from pyro.distributions.torch import *` higher up in pyro/distributions/__init__.py so that `pyro.distributions.Binomial` refers to your new `Binomial`, or
2. blacklist `Binomial` from the loop in `pyro.distributions.torch`

Reply:
I think 2. might better. Will fix.